### ISSUE TITLE
`SPC SPC` doesn't do M-x in terminal?

### ISSUE THREAD

**Issue by NightMachinary** [syl20bnr/spacemacs#10625]
Is this expected behavior?

**Comment by sdwolfz** [comment]
No it is not. Just tried it and it works.

Please share some more details about your configuration (.spacemacs file. and emacs/spacemacs versions).

**Comment by NightMachinary** [comment]
@sdwolfz 
#### System Info :computer:
- OS: darwin
- Emacs: 27.0.50
- Spacemacs: 0.300.0
- Spacemacs branch: added-dvorak-right-handed-to-keyboard-layout-layer (rev. 90440e5d)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(fsharp windows-scripts javascript xkcd spotify twitter search-engine vimscript
        (elfeed :variables rmh-elfeed-org-files
                (list "~/.emacs.d/private/feeds/elfeed1.org"))
        (rust :variables rust-format-on-save t)
        (haskell :variables haskell-enable-hindent t)
        semantic lsp
        (c-c++ :variables c-c++-enable-clang-support t)
        html python chrome
        (scala :variables scala-auto-insert-asterisk-in-comments t scala-use-unicode-arrows t scala-auto-insert-asterisk-in-comments t)
        (evil-snipe :variables evil-snipe-enable-alternate-f-and-t-behaviors t)
        autohotkey kotlin helm
        (auto-completion :variables auto-completion-enable-help-tooltip t auto-completion-enable-snippets-in-popup t)
        emacs-lisp
        (ibuffer :variables ibuffer-group-buffers-by 'modes)
        coq emoji
        (unicode-fonts :variables unicode-fonts-force-multi-color-on-mac t)
        git markdown org
        (shell :variables shell-default-height 30 shell-default-position 'bottom)
        spell-checking
        (syntax-checking :variables syntax-checking-enable-tooltips t)
        (elm :variables elm-sort-imports-on-save t elm-format-on-save t)
        version-control osx scheme
        (clojure :variables clojure-enable-fancify-symbols t))
```
- System configuration features: RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS


#### Backtrace :paw_prints:
```
<<BACKTRACE IF RELEVANT>>
```

**Comment by NightMachinary** [comment]
![image](https://user-images.githubusercontent.com/36224762/39305979-1db94a8c-4974-11e8-87e4-e0d3f434fd9e.png)
Here's a screenshot.

**Comment by sdwolfz** [comment]
Hmm weird, I have the exact setup as you right now, except the layers. I'd suggest you comment out parts of your `dotspacemacs/user-config` until you find the place that's causing problem.

**Comment by jsoo1** [comment]
@NightMachinary did you ever get this fixed?

**Comment by NightMachinary** [comment]
No, I don’t use the terminal that much.

**Comment by JustAPerson** [comment]
I get this happening in gui mode. `SPC SPC` doesn't work. Only started after I switched to `develop` branch.

Bisecting my config, setting one of these variables causes the problem `org-agenda-files`, `org-directory`, `org-default-notes-file`.

Sounds ridiculously unrelated, but commenting out those three (and at this point I don't care which) fixes the problem next time I restart emacs.

**Comment by jsoo1** [comment]
@JustAPerson do you have your org config wrapped in `(with-eval-after-load ‘org ...`?

**Comment by nfedyashev** [comment]
@JustAPerson you're a life saver!

If not this, I would not be able to use my GUI spacemacs anymore.

My context:
@ develop branch
Debian

Same spacemacs config that I used everywhere - GUI spacemacs on OSX, terminal spacemacs on linux & OSX.

Today I've tried to use it on a fresh MX Linux in GUI mode with the same config.

SPC SPC doesn't work anymore - instead, it switches to search mode and tries to find results as you continue to type after SPC SPC.

That's what I did and it solved(kind of) my issue:

comment out `(setq org-agend-files` block. Just that. I don't have org-directory or ord-default-notes-file settings in my config.

@JustAPerson thanks again!
